README.md

```diff
@@ -27,7 +27,13 @@ logo by [@nwlandas](https://twitter.com/nwlandas)
 
 ## Installation
 
-_it is in development and not yet finished_
```

Comment:
why did you remove this?
This is for non tech people who want to try fosscord and there is not a beta version yet.